src/WebJobs.Script/Config/FunctionsHostingConfigOptions.cs

```diff
@@ -147,6 +147,22 @@ internal bool EnableOrderedInvocationMessages
             }
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating whether to ignore the TestData property during read and write operations of functions metadata.
+        /// </summary>
+        internal bool EnableTestDataSuppression
```

Comment:
I realize this follows the convention of other properties here, but we should be using ‘Is<functionality>Enabled’

Reply:
I switched to `IsTestDataSuppressionEnabled`